sixtyfps_runtime/rendering_backends/gl/Cargo.toml

```diff
@@ -50,13 +50,15 @@ ttf-parser = "0.12.0"
 winit = { version = "0.25", default-features = false }
 glutin = { version = "0.27", default-features = false }
 glow = { version = "0.10.0", default-features = false }
-font-kit = { version = "0.10", features = ["loader-freetype"] }
+font-kit = { version = "0.10", features = [] }
 fontdb = { version = "0.5.1", features = ["fs"] }
 
-# Require font-config from the system. Issue #88 indicates that the copy provided by servo-fontconfig may be incompatible
-# with distros at times.
 [target.'cfg(not(any(target_family = "windows", target_os = "macos", target_os = "ios", target_arch = "wasm32")))'.dependencies]
```

Comment:
I'm wondering if this shouldn't be converted to a positive check instead

Reply:
Hmmm what would be the positive variant? linux/freebsd/netbsd/dragonfly/openbsd looks long :-)